csvspider.py

```diff
@@ -0,0 +1,42 @@
+import csv
+from datetime import datetime
+
+from scrapy import Request, Spider
+from six import text_type
+
+
+class CSVSpider(Spider):
+    """Visits the URLs from the specified column of the specified CSV file.
+
+    If you specify a protocol, column values are interpreted as domains, and
+    the specified protocol is used for requests.
+    """
+    name = 'csv'
+
+    custom_settings = {
+        'CONCURRENT_REQUESTS': 100,  # https://docs.scrapy.org/en/latest/topics/broad-crawls.html#increase-concurrency
+        'DUPEFILTER_DEBUG': True,
+        'HTTPERROR_ALLOW_ALL': True,
+        'REDIRECT_ENABLED': False,
```

Comment:
Why do we want to disable it?

Reply:
I’m not sure what my thinking was. Since it can be overriden, I see no issue in removing the setting.